### DISCUSSION TITLE
死水还阴？

### DISCUSSION THREAD

**Discussion by Crowinthewind** [share121/inter-knot#1741] [其他]
![7adf6f7c29c906b1cdba64de6a4120aab3285522](https://github.com/user-attachments/assets/b57f3717-f282-4393-95a3-a2e096acdb37)
不是，死水还阴啊？差不多得了，波风水门[秽土转生]大人怎么你了之后忘了，中间忘了，后面也忘了，总之就是闪光，你捉不到。你评价了死水对吧？我个人觉得波风水门【秽土转生】真的是最需要操作的忍者了。我对操作的定义就是这件事你知道怎么做但是你做不到。在目前的所有忍者中只有死水能让我有这种不说别的，被先手了替身2A插标闪走，这套操作没玩个几百把是真的做不到的，90%的人想做到这套操作都会发现自己2A根本插不出标来，训练营和决斗场的心理压力是不一样的，你在训练营就算能百分百做到2A插标，决斗场也90%的情况是2A插不出来。我玩骨架白面具破面小土豆，看到对面玩死水，就知道自己低人一等了，对面换位金身博弈我也是心服口服的，抓不到对面只能说我玩不明白死水，死水玩家摸头发表情我也是羞愧难当的，毕竟玩死水的绝对是全火影最需要操作的，我这种玩狗脑忍者的不能去攻击操作忍者的。我以后在火影忍者手游里，我以后再也不敢说秽土水门这个忍者的任何坏话，我承认死水这个忍者确实是要技术的，cd久的，阳间的，老忍者的，没抓取的，没反手的，被起手玩不了的，没有通灵秘卷t3的，伤害低的，要博弈的，z闪是难触发的，非常极端的，会被骨架克制的，打不过儿子黄猿的，必须加强，被暴打的心服口服的，求求你们不要再攻击我了，我不想在这个世界消失

**Comment by mur-cat** [comment]
这个委托我接了

**Comment by Crowinthewind** [comment]
这个委托我接了

**Comment by fu-fu404** [comment]
@~  叮，抱歉您的句子当中可能含有『水』字🤓，我可以认为您对死水进行了评价，已触发关键词，请您对我们的工作进行谅解🤓👉安排！
你刚刚是评价了死水是吗？
我认为死水是一个极其需要尊重的老角色，因为他在火影忍者手游存在的这八年内，给这个游戏带来了巨大的热度，如果没有死水，火影忍者手游可能就没有今天，所以死水应该是火影手游全体尊重的对象，死水不但制作精良，而且特别需要操作的手法，如果段位没有3000分玩这个忍者就不能发挥他的实力，一但不能完全发挥实力，这个忍者就是无比鸡肋的忍者，此时需要的就是意识和手法来弥补对这个角色的操作不足，当技术大成时即可打出号称游戏最需要操作的玩法，人们都称为z字闪。这个忍者真的没有你们想的那么简单，不仅仅是在游戏中，在动漫中死水也是一个沉稳伟大的父亲，虽然没给鸣人一个完整的童年，不过他在战场上，在村子中，都是耀眼的领袖，就和游戏中需要操作的阳间战神一样，黄色闪光，你抓不住

**Comment by phaeton66** [comment]
怎么还有村里人

**Comment by Yitaruose-BaiXu** [comment]
这个村里人的委托我接了

**Comment by sishui-buyin** [comment]
就是，死水还印啊

**Comment by ffewhawef** [comment]
这个委托我植物人接了
